api/src/controllers/collection-controller.js

```diff
@@ -157,7 +156,7 @@ const updateCollection = async function(incomingObj, collectionId, displayName)
   const CollectionBCMI = mongoose.model(RECORD_TYPE.CollectionBCMI._schemaName);
 
   const sanitizedObj = PutUtils.validateObjectAgainstModel(CollectionBCMI, incomingObj);
-  if (!sanitizedObj || sanitizedObj === {}) {
+  if (!sanitizedObj || Object.keys(sanitizedObj).length === 0) {
```

Comment:
If you want to shorten this further you can even replace `Object.keys(sanitizedObj).length === 0` with `!Object.keys(sanitizedObj).length`

Reply:
@pinkyandthekane That is a neat addition, But I think it hits the edge of readability at that level.